hello-world/nodejs-express-app/list-instances.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Sample code to get list of instances in particular region, compartment.
+ * You may still add more select criteria in ListInstancesRequest like compartmentId.
+ * 
+ * Sample Output (Masked):
+ * Instance: ocid1.instance.oc1.iad.anu...xru2a
+ */
+
+const common = require("oci-common");
+const core = require("oci-core");
+
+const adp = new common.ConfigFileAuthenticationDetailsProvider();
+
+const region = common.Region.US_ASHBURN_1;
```

Comment:
Use a different way of handling the region (variable, placeholder, command line parameter, environment variable, etc.).

Reply:
Yes. its now taken from environment variable. However, this is the default value. It can be overridden via query parameter. You may check below code
```
        computeClient.region = region;
        if(req.query.region)
            computeClient.region = req.query.region;
```